src/ConfigUpdaterResult.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\commerce;
+
+/**
+ * Result object for displaying configuration update results.
+ */
+class ConfigUpdaterResult implements ConfigUpdaterResultInterface {
+
+  /**
+   * Array of configuration names that failed to update.
+   *
+   * @var \Drupal\Component\Render\MarkupInterface[]
+   */
+  protected $failed = [];
+
+  /**
+   * Array of configuration names that succeeded to update.
+   *
+   * @var \Drupal\Component\Render\MarkupInterface[]
+   */
+  protected $succeeded = [];
+
+  /**
+   * ConfigUpdaterResult constructor.
+   *
+   * @param array $failed
+   *   An array of configuration names that failed to update.
+   * @param array $succeeded
+   *   An array of configuration names that succeeded to update.
+   */
+  public function __construct(array $failed, array $succeeded) {
```

Comment:
Wouldn't it make more sense for $succeeded to go first (in both the constructor and the methods)?